--- ispconfig3_install/interface/web/admin/lib/module.conf.php	2012-01-04 17:47:44.000000000 +0100
+++ ispconfig3_install2/interface/web/admin/lib/module.conf.php	2014-05-28 11:43:46.734485705 +0200
@@ -222,4 +222,33 @@
 	}
 }
 
-?>
\ No newline at end of file
+
+
+
+
+//* SOGo config options.
+if ($app->auth->get_client_limit($app->auth->get_user_id(), 'maildomain') != 0) {
+    $items = array();
+    $items[] = array('title' => 'Config',
+        'target' => 'content',
+        'link' => 'admin/sogo_config.php',
+        'html_id' => 'mail_sogo_config');
+
+    $items[] = array('title' => 'Domains',
+        'target' => 'content',
+        'link' => 'mail/mail_sogo_domain_list.php',
+        'html_id' => 'mail_sogo_domain_list');
+
+    $items[] = array('title' => 'Thunderbird Plugins',
+        'target' => 'content',
+        'link' => 'admin/sogo_thunderbird_plugins.php',
+        'html_id' => 'mail_sogo_config');
+
+    $module['nav'][] = array('title' => 'SOGo',
+        'open' => 1,
+        'items' => $items);
+}
+
+
+
+?>
