--- ispconfig2/interface/web/mail/lib/module.conf.php	2014-05-19 18:45:04.000000000 +0200
+++ ispconfig/interface/web/mail/lib/module.conf.php	2014-05-19 18:56:02.000000000 +0200
@@ -204,4 +204,18 @@
 								'open' 	=> 1,
 								'items'	=> $items);
 }
+
+
+
+//* SOGo config option.
+if ($app->auth->get_client_limit($app->auth->get_user_id(), 'maildomain') != 0) {
+    $items = array();
+    $items[] = array('title' => 'SOGo Domains',
+        'target' => 'content',
+        'link' => 'mail/mail_sogo_domain_list.php',
+        'html_id' => 'mail_sogo_domain_list');
+    $module['nav'][] = array('title' => 'SOGo',
+        'open' => 1,
+        'items' => $items);
+}
 ?>
\ No newline at end of file
